src/server/scripts/Commands/cs_modify.cpp

```diff
@@ -123,7 +128,163 @@ class modify_commandscript : public CommandScript
         return true;
     }
 
-    //Edit Player HP
+    // Edit Player Strength
+    static bool HandleModifyStrengthCommand(ChatHandler* handler, char const* args)
+    {
+        if (!*args)
+            return false;
+
+        int32 stat = atoi((char*)args);
+
+        if (stat <= 0 )
+        {
+            handler->SendSysMessage(LANG_BAD_VALUE);
+            handler->SetSentErrorMessage(true);
+            return false;
+        }
+
+        Player *chr = handler->GetSelectedPlayerOrSelf();
+        if (chr == nullptr)
+        {
+            handler->SendSysMessage(LANG_NO_CHAR_SELECTED);
+            handler->SetSentErrorMessage(true);
+            return false;
+        }
+
+        handler->PSendSysMessage("You change strength of %s to %u.", chr->GetName().c_str(), stat);
+
+        chr->HandleStatFlatModifier(UNIT_MOD_STAT_STRENGTH, BASE_VALUE, float(stat), true);
+        chr->UpdateStatBuffMod(STAT_STRENGTH);
+
+        return true;
+    }
+
+    // Edit Player Agility
+    static bool HandleModifyAgilityCommand(ChatHandler* handler, char const* args)
```

Comment:
I'd very much like to avoid redundancy here and use a common function for them.
Could be something like `HandleModifyStatCommand(ChatHandler* handler, std::string statName, UnitMods modStat, Stats stat)`
For example with agility it would be called like this: 
`HandleModifyStatCommand(handler, "agility", UNIT_MOD_STAT_AGILITY, STAT_AGILITY);`